ocs_ci/helpers/performance_lib.py

```diff
@@ -37,3 +38,51 @@ def write_fio_on_pod(pod_obj, file_size):
         pod_obj, file_path
     ), f"File {file_name} does not exist"
     logger.info(f"File {file_name} exists in {pod_obj.name}.")
+
+
+def run_command(cmd, timeout=600, out_format="string", **kwargs):
+    """
+    Running command on the OS and return the STDOUT & STDERR outputs
+    in case of argument is not string or list, return error message
+
+    Args:
+        cmd (str/list): the command to execute
+        timeout (int): the command timeout in seconds, default is 10 Min.
+        out_format (str): in which format to return the output: string / list
+        kwargs (dict): dictionary of argument as subprocess get
+
+    Returns:
+        list or str : all STDOUT and STDERR output as list of lines, or one string separated by NewLine
+
+    """
+
+    if isinstance(cmd, str):
+        command = cmd.split()
+    elif isinstance(cmd, list):
+        command = cmd
+    else:
+        return "Error in command"
+
+    for key in ["stdout", "stderr", "stdin"]:
+        kwargs[key] = subprocess.PIPE
+
```

Comment:
@Avilir What will happen if stdout/stderr/stdin values will be already passed to this function in kwargs? Wouldn't it be overwritten here by this code?

Reply:
that is correct, but the purpose of this function is to capture and return all this data.
if (for instance) you want to send it to file, you need to use the subprocess in a different way.